Antidote/AppearanceManager.m

```diff
@@ -221,6 +221,11 @@ - (UIColor *)bubbleIncomingColorForScheme:(AppearanceManagerColorscheme)scheme
     }
 }
 
+- (UIColor *)callRedColor
+{
+    return [UIColor colorWithRed:0.86 green:0.24 blue:0.24 alpha:1.0];
+}
```

Comment:
This is good idea to move colors here. You can move other call related colors too (except for standard colors like `clear`, etc)